<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaPrincipal</class>
 <widget class="QMainWindow" name="JanelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista de Tarefas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/pngwing.com.png</normaloff>../../../Downloads/pngwing.com.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color:rgb(30, 30, 30);
}





</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>501</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="2" colspan="2">
      <widget class="QPushButton" name="btn_concluida">
       <property name="text">
        <string>Marcar como conclu√≠da</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="3" colspan="2">
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Digite aqui...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn_add">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color:rgb(34, 63, 14);
	color:rgb(0, 172, 83);
}

</string>
       </property>
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btn_remove">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color:rgb(46, 36, 44);
	color:rgb(176, 114, 121);
}</string>
       </property>
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="btn_salvartarefas">
       <property name="text">
        <string>Salvar tarefas</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QListWidget" name="listWidget">
       <property name="font">
        <font>
         <family>Yu Gothic</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget
{
	  background-color:rgb(30, 30, 30);
	color:rgb(255, 255, 255);
	border:none;
}

QListWidget::indicator:unchecked {
    background-color: rgb(30, 30, 30);
    color:white;     
    border: 1px solid white; 
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>181</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color:rgb(40, 40, 40);
}


</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_username2">
          <property name="font">
           <font>
            <family>Segoe UI Emoji</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel
{
	color:white
}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_points2">
          <property name="font">
           <font>
            <family>Segoe UI Emoji</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel
{
	color:white
}</string>
          </property>
          <property name="text">
           <string> TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lbl_icone">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="status">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Emoji</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color:rgb(8, 107, 28);
}</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>lbl_icone</zorder>
    <zorder>gridLayoutWidget_3</zorder>
    <zorder>status</zorder>
   </widget>
   <widget class="QLabel" name="label_add_tarefa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Tarefas</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
